packages/preview/lib/mail.js

```diff
@@ -55,7 +99,13 @@ const getGlobalMergeVars = () => ([
   { name: 'LINK_IMPRINT',
     content: `${FRONTEND_BASE_URL}/impressum`
   },
+  { name: 'LINK_IMPRESSUM',
+    content: `${FRONTEND_BASE_URL}/impressum`
+  },
   { name: 'LINK_PROJECTR',
     content: 'https://project-r.construction/'
+  },
+  { name: 'LINK_PROJECTR_NEWS',
+    content: 'https://project-r.construction/news'
```

Comment:
These urls should all go to translations, as they are heavily specific to our environment.

Reply:
Not sure how I can manage to make it agnostic. Any hints here?